src/test/java/uk/gov/companieshouse/confirmationstatementapi/client/OracleQueryClientTest.java

```diff
@@ -181,4 +184,32 @@ void testGetShareholderNotOkStatusResponse() {
         assertTrue(serviceException.getMessage().contains(companyNumber));
         assertTrue(serviceException.getMessage().contains(HttpStatus.SERVICE_UNAVAILABLE.toString()));
     }
+
+    @Test
+    void testIsConfirmationStatementPaid() throws ServiceException {
+        ResponseEntity<Boolean> response = ResponseEntity.status(HttpStatus.OK).body(Boolean.TRUE);
```

Comment:
I don't think we are allowed to return just a value in the response that isn't attached to a field that indicates what it represents. As far as I know all CHS api return field: value  in the response. I'll tag @ICoxCH and @sroberts2 for advice but looks like it's something I missed in the review of the oracle-query-api paid endpoint.

Reply:
I have tested this and it works as expected. Not allowed by who?

Reply:
I'll raise it separately and get others views as it doesn't really affect this pr , i wasn't sure if the oracle-query-api response was inline with the other CHS apis ie. whether the responses need to be wrapped in a json object or if we can return the primitive value